openxr/src/session.rs

```diff
@@ -428,6 +428,77 @@ impl<G: Graphics> Session<G> {
             Ok(Swapchain::from_raw(self.clone(), out))
         }
     }
+
+    #[inline]
+    /// Create a [`Passthrough`].
+    ///
+    /// Requires [`XR_FB_passthrough`].
+    ///
+    /// [`XR_FB_passthrough`]: https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XR_FB_passthrough
+    /// [`Passthrough`]: passthrough/struct.Passthrough.html
+    pub fn create_passthrough(&self, flags: PassthroughFlagsFB) -> Result<Passthrough> {
+        Passthrough::create(self, flags)
+    }
+
+    #[inline]
+    /// Pause a [`Passthrough`].
+    ///
+    /// Requires [`XR_FB_passthrough`].
+    ///
+    /// [`XR_FB_passthrough`]: https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XR_FB_passthrough
+    /// [`Passthrough`]: passthrough/struct.Passthrough.html
+    pub fn start_passthrough(&self, passthrough: &mut Passthrough) -> Result<()> {
+        passthrough.start(&self.inner)
+    }
+
+    #[inline]
+    /// Pause a [`Passthrough`].
+    ///
+    /// Requires [`XR_FB_passthrough`].
+    ///
+    /// [`XR_FB_passthrough`]: https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XR_FB_passthrough
+    /// [`Passthrough`]: passthrough/struct.Passthrough.html
+    pub fn pause_passthrough(&self, passthrough: &mut Passthrough) -> Result<()> {
+        passthrough.pause(&self.inner)
+    }
+
+    #[inline]
+    /// Create a [`PassthroughLayer`].
+    ///
+    /// Requires [`XR_FB_passthrough`].
+    ///
+    /// [`XR_FB_passthrough`]: https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XR_FB_passthrough
+    /// [`PassthroughLayer`]: passthrough/struct.PassthroughLayer.html
+    pub fn create_passthrough_layer(
+        &self,
+        passthrough: &Passthrough,
+        flags: PassthroughFlagsFB,
+        purpose: PassthroughLayerPurposeFB,
+    ) -> Result<PassthroughLayer> {
+        PassthroughLayer::create(self, passthrough, flags, purpose)
+    }
+
+    #[inline]
+    /// Pause a [`PassthroughLayer`].
+    ///
+    /// Requires [`XR_FB_passthrough`].
+    ///
+    /// [`XR_FB_passthrough`]: https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XR_FB_passthrough
+    /// [`PassthroughLayer`]: passthrough/struct.PassthroughLayer.html
+    pub fn pause_passthrough_layer(&self, layer: &mut PassthroughLayer) -> Result<()> {
+        layer.pause(&self.inner)
+    }
+
+    #[inline]
+    /// Resume a [`PassthroughLayer`].
+    ///
+    /// Requires [`XR_FB_passthrough`].
+    ///
+    /// [`XR_FB_passthrough`]: https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#XR_FB_passthrough
+    /// [`PassthroughLayer`]: passthrough/struct.PassthroughLayer.html
+    pub fn resume_passthrough_layer(&self, layer: &mut PassthroughLayer) -> Result<()> {
+        layer.resume(&self.inner)
+    }
```

Comment:
All of these save for `create_passthrough` should be methods on the appropriate new struct, not on `Session`.